package.json

```diff
@@ -18,7 +18,8 @@
   "homepage": "https://github.com/treasure-data/eslint-plugin-td",
   "license": "Apache-2.0",
   "dependencies": {
-    "import-modules": "^1.1.0"
+    "import-modules": "^1.1.0",
+    "jsonpath": "^1.0.2"
```

Comment:
This appears to be similar to `lodash.get`. I think we should use lodash to keep consistent with our other codebases